src/components/Thread.vue

```diff
@@ -256,10 +256,7 @@ export default {
 
 <style lang="scss">
 #mail-message {
-	flex-grow: 1;
-	max-height: calc(100vh - 50px);
 	margin-bottom: 30vh;
```

Comment:
why is this needed?

Reply:
The margin was added deliberately to be able to scroll past the last thread. I don't remember anymore but I guess it was feedback from a design review.